### PR TITLE
Add GOF space

### PR DESCRIPTION
**Pull request by aisinoluhua**

### CODE REVIEW

**Comment by aisinoluhua** [comment]
add GOF space

File: spaces/golff/skin.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.yam {
```
Comment (e-asia) [inline spaces/golff/skin.scss:1]: The css selector should similar to the space key.
```suggestion
.golff {
```

File: spaces/golff/index.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "key": "gof",
```
Comment (e-asia) [inline spaces/golff/index.json:2]: The key of your space must also be the folder name and skin class name.
```suggestion
  "key": "golff",
```

**Review by e-asia — CHANGES_REQUESTED** [review]

**Comment by aisinoluhua** [comment]
The space key and skin class name have been modified

File: spaces/golff/index.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  "address": "0x488E0369f9BC5C40C002eA7c1fe4fd01A198801c",
+  "token": "0x488E0369f9BC5C40C002eA7c1fe4fd01A198801c",
+  "core": [
+    "0x424abfc7c0Defb02447D621Bf0f6eF80eD5C01fB"
+  ],
+  "min": 100,
+  "invalid": [
+    "0x424abfc7c0Defb02447D621Bf0f6eF80eD5C01fB"
+  ]
+}
```
Comment (e-asia) [inline spaces/golff/index.json:15]: We have updated the format in `index.json`. 
Please add 2 new parameters `chainId` and `decimals`.
You can refer to the documentation that has been completed. https://docs.snapshot.page/guides/create-a-space#3-change-your-space-metadata

**Review by e-asia — CHANGES_REQUESTED** [review]

File: spaces/golff/index.json (hunk 3 above)
Comment (aisinoluhua) [reply spaces/golff/index.json:15]: > We have updated the format in `index.json`.
> Please add 2 new parameters `chainId` and `decimals`.
> You can refer to the documentation that has been completed. https://docs.snapshot.page/guides/create-a-space#3-change-your-space-metadata

like this?
{
  "key": "golff",
  "symbol": "GOF",
  "name": "Golff Finance",
  "defaultView": "core",
  "address": "0x488E0369f9BC5C40C002eA7c1fe4fd01A198801c",
  "token": "0x488E0369f9BC5C40C002eA7c1fe4fd01A198801c",
  "chainId":1,
  "decimals":18,
  "core": [
    "0x424abfc7c0Defb02447D621Bf0f6eF80eD5C01fB"
  ],
  "min": 100,
  "invalid": [
    "0x424abfc7c0Defb02447D621Bf0f6eF80eD5C01fB"
  ]
}

Comment (e-asia) [reply spaces/golff/index.json:17]: @aisinoluhua Yes 👍

**Review by e-asia — APPROVED** [review]

**Pull request merged**